toast/toast.css

```diff
@@ -0,0 +1,36 @@
+@import "../buttons/buttons.css";
+
+.container{
+    padding: 1rem;
+    gap: 2rem;
+}
+.component-container{
+    padding: 2rem 4rem 2rem 2rem;
+}
+.component-details{
+    padding: 1rem;
+}
+#myToast {
+   visibility: hidden;
+    position: fixed;
+    bottom: 0.5rem;
+    right: 40%;
+    background-color: var(--grey-color);
+    color: var(--white-color);
+    z-index: 1; 
+    padding: 2rem;
+  }
+
+.button{
+    background-color: var(--primary-color);
+    margin: 1rem;
+}
```

Comment:
changes: if this button class is specific to toast then write the class name as 'toast-button'. If this is common for all other components then keep this class in utility.css and import utility.css in all other HTML files using link tag in < head >